scripts/env-setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+################################################
+# Generates and sets environment variables for broker
+#
+################################################
+
+# Set the network in the .env file based on user input
+echo "Enter the network: 1 for MainNet, 2 for TestNet"
+read ADDR
+if [ $ADDR = '1' ]; then
+  sed -i "" -e "s/.*NETWORK.*/NETWORK=mainnet/" .env
```

Comment:
do we want to first copy over a sample .env file? Or should that happen somewhere else? Or maybe that only happens if the .env file doesn't exist?

Reply:
We have them do it manually but i guess we could do it based on the network they choose 🤷‍♀️